common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -178,8 +178,26 @@ public final List<Package> importAllPackages(final List<Package> vroPackages, fi
 				String backupFilePath = this.createBackupFilePath(pkg, currentDateTimeString, backupFilesDirectory);
 
 				try {
-					pkg.setFilesystemPath(backupFilePath);
-					restClient.exportPackage(pkg, dryrun, exportConfigAttributeValues, exportConfigSecureStringValues);
+					System.out.println("Package for back up: " + pkg.getFQName());
+
+					List<Package> samePackagesInDest = new ArrayList<Package>();
+					for (int i = 0; i < destinationEndpointPackages.size(); i++) {
+						String currentVroPackageName = destinationEndpointPackages.get(i).getName();
+
+						if (currentVroPackageName.equals(pkg.getName())) {
+							samePackagesInDest.add(destinationEndpointPackages.get(i));
+						}
+					}
+
+					if (samePackagesInDest.size() > 0) {
+						Package highestVersionPac = this.getHighestVersion(samePackagesInDest);
+						highestVersionPac.setFilesystemPath(backupFilePath);
+						restClient.exportPackage(highestVersionPac, dryrun, exportConfigAttributeValues, exportConfigSecureStringValues);
+					
+					} else {
+						System.out.println("Package with name: " + pkg.getName() + " does not exist in vRO and backup is skipped.");
```

Comment:
Can you please user the logger object to send information instead of system.out?